### PR TITLE
Make `items` an include on works in the API

### PR DESCRIPTION
**Pull request by alexwlchan**
### What is this PR trying to achieve?

As the description says. This is mentioned as a requirement on #572.

### Who is this change for?

People who like to stick to the requirements.

### Have the following been considered/are they needed?

- [ ] Reviewed by @jtweed
- [ ] Updated the Swagger documentation
- [ ] Deployed new versions

### CODE REVIEW

File: api/src/main/scala/uk/ac/wellcome/platform/api/models/WorksIncludes.scala [hunk 1]
```diff
@@ -31,7 +32,8 @@ case object WorksIncludes {
     if (unrecognisedIncludes.length == 0) {
```
Comment (alicefuzier) [inline api/src/main/scala/uk/ac/wellcome/platform/api/models/WorksIncludes.scala]: minor: can we use `.isEmpty` here?

File: api/src/test/scala/uk/ac/wellcome/platform/api/ApiWorksTest.scala [hunk 2]
```diff
@@ -513,8 +500,7 @@ class ApiWorksTest
                           |     "title": "${work1.title}",
                           |     "creators": [],
                           |     "subjects": [ ],
-                          |     "genres": [ ],
-                          |     "items": [ ]
+                          |     "genres": [ ]
```
Comment (alicefuzier) [inline api/src/test/scala/uk/ac/wellcome/platform/api/ApiWorksTest.scala]: This test is about rendering items with no ids, but items have been taken out? Or am I missing something?

File: api/src/main/scala/uk/ac/wellcome/platform/api/models/WorksIncludes.scala (hunk 1 above)
Comment (alexwlchan) [reply api/src/main/scala/uk/ac/wellcome/platform/api/models/WorksIncludes.scala]: 👍

File: api/src/test/scala/uk/ac/wellcome/platform/api/ApiWorksTest.scala (hunk 2 above)
Comment (alexwlchan) [reply api/src/test/scala/uk/ac/wellcome/platform/api/ApiWorksTest.scala]: No, that’s on L194 and 210 – this test is

> should return matching results if doing a full-text search

**Comment by alexwlchan** [comment]
> Looks ok, but I don't see any test where the include=items is passed in? I think at least one of the ApiTests should test this case

Agreed, but I think there’s already a test for this. See:

> should be able to render an item with no canonicalId if the items include is present

**Comment by alicefuzier** [comment]
Uhm, I still think it should have its own test. The `should be able to render an item with no canonicalId if the items include is present` one was about the api not crashing if items have no id. It just now happens to require the include. If that is the only test for the include=items thing we achieve the code coverage, perhaps, but we kind of lose the documentation aspect of tests cause that is very specific to items with no id. The items include has nothing to do with that

**Comment by alexwlchan** [comment]
Ah, gotcha. Will change.

**Comment by alexwlchan** [comment]
Okay, latest couple of commits have a test that just covers the `?includes=items` identifier.

**Pull request merged**